Uzira.rte/Devices/Weapons/Boomstick/Boomstick.ini

```diff
@@ -1,168 +1,136 @@
 ///////////////////////////////////////////////////////////////////////
-// Hand Mortar Ammo
+// Boomstick Ammo
 
 
-AddAmmo = MOSRotating
-	PresetName = Hand Mortar Bomb
-	Mass = 1
-	Sharpness = 0.1
-	RestThreshold = -500
-	AirResistance = 0.008
-	AirThreshold = 60
-	Sharpness = 1
+AddParticle = MOSRotating
+	PresetName = Particle Uzira Boomstick
+	Mass = 4
+	LifeTime = 10000
+	RestThreshold = -1
+	GibAtEndOfLifetime = 1
 	HitsMOs = 1
-	GetsHitByMOs = 0
+	GetsHitByMOs = 1
 	SpriteFile = ContentFile
-		FilePath = Uzira.rte/Devices/Weapons/HandMortar/MortarBomb.png
-	ScriptPath = Uzira.rte/Devices/Weapons/HandMortar/MortarBomb.lua
```

Comment:
Why did you get rid of the random detonation time? I was a big fan of that, made it feel more unreliable and interesting.
If this is undone, don't forget the description change!

Reply:
The last thing undead units need imo is a weapon that puts themselves in danger, this thought was purely from an AI perspective

Skellies are unreliable enough due to their durability, not "unreliable" as a general description (imo)

Reply:
IMO my perspective is that we should make them a challenge by just having a quadrillion of them on screen at once. As such I love flavour things like this

Reply:
Fair point 4zK, but I personally agree with causeless on this. Especially with higher performance, swarms of zombies is fun, and having their bombs exploding randomly seems fun to me.

If you're really strongly against this though, I'm okay with keeping not having the randomness. It's up to you :)